python/hsfs/feature_view.py

```diff
@@ -0,0 +1,379 @@
+#
+#   Copyright 2022 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+from datetime import datetime
+from typing import Optional, Any, Dict, List
+
+import humps
+
+from hsfs import tag, util, training_dataset_feature
+from hsfs.constructor import query
+from hsfs.core import (
+    feature_view_engine, statistics_engine,
+    transformation_function_engine, vector_server
+)
+from hsfs.transformation_function import TransformationFunction
+
+
+class FeatureView:
```

Comment:
This PR only deals with API for Feature View metadata and online vector serving.